build.py

```diff
@@ -39,33 +50,48 @@ def _parse_args():
         type=Path,
         # We set the default programmatically as it needs to take into account whether we're cross-compiling
         help="Path to the build directory. Defaults to 'build/<target platform>'. "
-             "The build configuration will be a subdirectory of the build directory. e.g. build/Linux/Debug",
+        "The build configuration will be a subdirectory of the build directory. e.g. build/Linux/Debug",
     )
 
     parser.add_argument(
         "--config",
         default="RelWithDebInfo",
         type=str,
         choices=["Debug", "MinSizeRel", "Release", "RelWithDebInfo"],
-        help="Configuration to build.")
+        help="Configuration to build.",
+    )
 
     # Build phases.
     parser.add_argument("--update", action="store_true", help="Update makefiles.")
     parser.add_argument("--build", action="store_true", help="Build.")
     parser.add_argument("--test", action="store_true", help="Run tests.")
     parser.add_argument(
-        "--clean", action="store_true", help="Run 'cmake --build --target clean' for the selected config/s."
+        "--clean",
+        action="store_true",
+        help="Run 'cmake --build --target clean' for the selected config/s.",
     )
 
-    parser.add_argument("--skip_tests", action="store_true", help="Skip all tests. Overrides --test.")
-    parser.add_argument("--skip_wheel", action="store_true", help="Skip building the Python wheel.")
-    parser.add_argument("--skip_csharp", action="store_true", help="Skip building the C# API.")
+    parser.add_argument(
+        "--skip_tests", action="store_true", help="Skip all tests. Overrides --test."
+    )
+    parser.add_argument(
+        "--skip_wheel", action="store_true", help="Skip building the Python wheel."
+    )
+    parser.add_argument(
+        "--skip_csharp", action="store_true", help="Skip building the C# API."
+    )
```

Comment:
Is there a requirement for this sort of formatting? personally I think it reduces readability in cases like this so I'm not a fan. IIRC it's the `black` formatter. I know we use it in ORT but don't in extensions. 

I'm fine with using formatters for consistency though - I'd just prefer to use things that don't reduce readability when the developer has made specific choices about where to put line breaks. FWIW this has added ~100 lines of whitespace to the existing script.

Reply:
make sense - reverted some of the formatting changes.